ember_debug/render-debug.js

```diff
@@ -35,9 +36,37 @@ export default EmberObject.extend(PortMixin, {
   },
 
   sendAdded(profiles) {
+    if (this.shouldHighlightRender) {
+      profiles.forEach(profile => {
+        this._hightLightNode(profile);
+      })
+    }
     this.sendMessage('profilesAdded', { profiles });
   },
 
+  _hightLightNode({ viewGuid, children }) {
+    const hasChildren = children?.length > 0;
+    if (viewGuid) {
+      this._createOutline(viewGuid, hasChildren)
+    }
+    if (hasChildren) {
+      children.forEach(childNode => {
+        this._hightLightNode(childNode);
+      })
+    }
+  },
+
+  _createOutline(viewGuid, hasChildren) {
+    const element = document.getElementById(viewGuid);
+    if (element) {
+      const outline = element.style.outline;
+      element.style.outline  = `${hasChildren ? '0.5' : '1'}px solid ${hasChildren ? 'blue' : 'red'}`;
+      setTimeout(()=> {
```

Comment:
could we clear timeout in case of more than one rerender will happen in 1s?

Reply:
I think it is ok and normal if the highlights of different components happen at the same time. React dev tool has the same behavior which seems to be pretty good.
However, you remind me that we can have some special highlight if the same component got re-rendered within short time, 300ms for example. I can add that feature in the future change.

Reply:
Great point, I will make the change.

Reply:
changed.